modules/virt-creating-vm-instancetype.adoc

```diff
@@ -6,30 +6,29 @@
 [id="virt-creating-vm-instancetype_{context}"]
 = Creating a virtual machine from an instance type
 
-[IMPORTANT]
-====
-This is a Developer Preview feature only. Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. Do not use Developer Preview features for production or business-critical workloads. Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
-====
-
 You can use common instance types to install and edit customized instance types and preferences to create a virtual machine (VM).
 
-.Procedure
+:FeatureName: Creating a virtual machine from an instance type
+include::snippets/technology-preview.adoc[]
 
-. In the web console, navigate to the *InstanceTypes* tab in one of the following ways:
+.Procedure
 
-+
-* Click *Virtualization* -> *Catalog* -> *InstanceTypes*.
-* Click *Virtualization* -> *VirtualMachines* and select *From volume* in the *Create* dropdown menu.
+. In the web console, navigate to *Virtualization* -> *Catalog* -> *InstanceTypes*.
 
 . Select a volume to boot from.
 
-. Select an 'InstanceType'.
+[NOTE]
```

Comment:
You should add a `+` before the [NOTE]. Without the `+`, it is breaking up the flow and starting a new numbering sequence with the following step.